src/cli/tc.rs

```diff
@@ -37,9 +42,57 @@ pub async fn run() -> Result<()> {
         Err(e) => Err(e),
       }
     }
+    Command::Init { file_path } => Ok(init(&file_path).await?),
   }
 }
 
+pub async fn init(file_path: &str) -> Result<()> {
+  let tailcallrc = resource_str!("assets/.tailcallrc.graphql");
+
+  let ans = Confirm::new("Do you want to add a file to the project?")
+    .with_default(false)
+    .prompt();
+
+  match ans {
+    Ok(true) => {
+      println!("{}", "Enter the file name:".yellow());
```

Comment:
```suggestion
      println!("{}", "Enter the file name:".yellow());
```
Can we use inquire here?